docs/reference.md

```diff
@@ -0,0 +1,327 @@
+# Reference for 'kubectl kubtest` plugin
```

Comment:
We can point to cobra generated CLI reference here -> https://github.com/kubeshop/kubtest/tree/main/docs/cli

Reply:
added links to these to the menu